ansible/roles/datadog/vars/main.yml

```diff
@@ -14,11 +14,11 @@ datadog_parsers:
     python_file: "{{ datadog_parsers_dest }}/nginx/timings.py"
     python_function: parse_nginx_counter
     hosts: "{{ groups['proxy'] }}"
-  - input: "{{ log_home }}/{{ inventory_hostname_short }}-commcarehq.couch.log"
+  - input: "{{ log_home }}/{{ localsettings.DEPLOY_MACHINE_NAME }}-commcarehq.couch.log"
```

Comment:
because we change which one use based on the env :/ https://github.com/dimagi/commcarehq-ansible/blob/master/ansible/roles/commcarehq/templates/localsettings.py.j2#L61